creator-node/test/users.test.js

```diff
@@ -159,60 +159,6 @@ describe('test Users', async function () {
       .expect(200)
   })
 
-  it('allows user login', async function () {
```

Comment:
is there already a test for the new challenge based login?

Reply:
yup